### ISSUE TITLE
Get real tweet message

### ISSUE THREAD

**Issue by Sequoya42** [twitterdev/twitter-api-typescript-sdk#93]
A tweet includes all the `@` of the reply, that don't appear on the website.

tldr: Tweet from API and actual tweet not the same text.

### Expected behavior
Having the actual text of the tweet.

**As concisely as possible, describe the expected behavior.**

If the tweet is a reply to @bob but the tweet is "Hello"
To not have the tweet.text as `@bob Hello`
Could be info put as meta. Because how to distingish if the user actually replies or tag or both?
Because the tweet could be `@bob Hello`.

And this (seems to be) impossible right now, without fetching the whole chain to see if it's actually in the tweet or come from the original post. Or simply a reply to someone, without explicitly writing it.
